File: src/main/java/cn/gov/forestry/common/domain/bo/SystemBuildInFieldEnum.java
package cn.gov.forestry.common.domain.bo;

import lombok.Getter;

public enum SystemBuildInFieldEnum {
    ID("id", 1),
    GEOMETRY("geometry", 2),
    GEOMETRY_BBOX("geometry_bbox", 2),
    GEOMETRY_AREA("geometry_area", 2),
    ORDER("order", 3)
    ;
    @Getter
    private String fieldName;

    private Integer fieldFilter;


    SystemBuildInFieldEnum(String fieldName, Integer fieldFilter) {
        this.fieldName = fieldName;
        this.fieldFilter = fieldFilter;
    }


}

File: src/main/python/cn/gov/forestry/common/domain/bo/system_build_in_field_enum.py
from enum import Enum


class SystemBuildInFieldEnum(Enum):
    ID = ("id", 1)
    GEOMETRY = ("geometry", 2)
    GEOMETRY_BBOX = ("geometry_bbox", 2)
    GEOMETRY_AREA = ("geometry_area", 2)
    ORDER = ("order", 3)

    def __init__(self, field_name: str, field_filter: int):
        self.fieldName = field_name
        self.fieldFilter = field_filter


__all__ = ["SystemBuildInFieldEnum"]
